runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -46,6 +45,33 @@ static std::unique_ptr<ExecutionContext> CTX = nullptr;
  */
 thread_local static RTDevice *RTD_PTR = nullptr;
 
+bool getModifiersAdjoint(const Modifiers *modifiers)
+{
+    return modifiers == nullptr ? false : modifiers->adjoint;
+}
+
+std::vector<QubitIdType> getModifiersControlledWires(const Modifiers *modifiers)
+{
+    return modifiers == nullptr ? std::vector<QubitIdType>()
```

Comment:
it should work, right?
```suggestion
    return modifiers ? std::vector<QubitIdType>()
```